### PR TITLE
added rating component

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Please review urgently.

### CODE REVIEW

**Review by sharmivijay — APPROVED** [review]
Code looks neat and try to use rem instead of pixels as it helps in responsiveness

File: Rating/rating.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const ratingStars = [...document.getElementsByClassName("rating__star")];
```
Comment (Snigdha17) [inline Rating/rating.js:1]: good destructuring

**Review by Snigdha17 — APPROVED** [review]
Change looks good to me. You can convert px to rems and merge

**Review by aamrapaliwandhre — APPROVED** [review]

File: Rating/rating.js (hunk 1 above)
Comment (MonikaJhingan) [reply Rating/rating.js:1]: thx

**Pull request merged**